projects/WTI-API/src/main/controllers/ContestController.java

```diff
@@ -566,7 +582,53 @@ public Response getStandings(
 					// we got the internal contest; pass it to the DefaultScoringAlgorithm and get back updated standings
 					try {
 						Properties props = internalContest.getContestInformation().getScoringProperties();
-						String xmlStandings = dsa.getStandings(internalContest, props, logger);
+
+						xlog(logger, "debug 22 scoreboard 2 props "+props);
+						if (props != null) {
+						       Set<Object> set = props.keySet();
+
+						        String[] keys = (String[]) set.toArray(new String[set.size()]);
+
+						        Arrays.sort(keys);
+								xlog(logger, "debug 22   Scoring Properties, there are " + keys.length + " keys");
+
+						        for (String string : keys) {
+									xlog(logger, "debug 22      " + string + "='" + props.get(string) + "'");
+						        }
+						}
+						
+						props = DefaultScoringAlgorithm.getDefaultProperties();
```

Comment:
The above line is (should be) a very temporary fix.  As a result, it should at _least_ have a comment that indicates
- it is temporary.
- the issue which reports the underlying problem for which this is a temporary workaround.

Reply:
imo,  an `if props == null`  should be added to that call.

Reply:
will add comment and conditional.

Reply:
> imo, an if props == null should be added to that call.

Add where?  In the current/PR code is there any possibility that the prop == null ?

Reply:
pushed new "temp" comment